src/test/scala/nl.knaw.dans.easy.bagstore/server/StoresServletSpec.scala

```diff
@@ -333,6 +333,17 @@ class StoresServletSpec extends TestSupportFixture
     }
   }
 
+  it should "fail when an inactive bag is requested with the wrong queryParams" in {
+    val bagId = BagId(UUID.fromString("01000000-0000-0000-0000-000000000001"))
+    inside(bagStore1.deactivate(bagId)) {
```

Comment:
I suppose this is a preparation for the actual test. Call fail when you get a failure in those cases.